timetabler/scheduler.py

```diff
@@ -32,7 +32,8 @@ def __init__(self, courses, session="2014W", terms=(1, 2),
         :type ssc_conn: SSCConnection
         """
         self.ssc_conn = SSCConnection() if ssc_conn is None else ssc_conn
-        self.courses = {c: self.ssc_conn.get_course(c, session, refresh=refresh)
+        self.courses = {c: self.ssc_conn.get_course(c, session, refresh=refresh,
```

Comment:
I should be; if I'm not, feel free to correct it haha.

Also, I don't follow 79 line length strictly all the time; if there is no real benefit to splitting a line and it's going **just over** I'll leave that be (because in this case, PEP8 is a suggestion, not a commandment).